### DISCUSSION TITLE
"Mutating statement" error

### DISCUSSION THREAD

**Discussion by Enphynity** [dbeaver/dbeaver#35098] [Q&A] [answered]
### Description

I am trying to create a view in a SQL Server database and I'm getting an error message that states "Mutating statement execution is prohibited in read-only connection".  I did not get this error prior to the recent update.  Is there some way to disable this as it is impacting my work.

### DBeaver Version

Community 24.1.4.202408041450

### Operating System

Windows 11

### Database and driver

SQL Server database
MS SQL Server / SQL Server driver

**Comment by ask9** [comment, marked as answer]
Open properties of your connection (right mouse click -> Edit connection) and uncheck the option: "Read-only connection" on the tab "General"

![изображение](https://github.com/user-attachments/assets/5303e883-8da5-4823-b302-aacd91acc648)

**Comment by Enphynity** [comment]
This worked, thank you
